src/BuildScriptGenerator/Node/NodeConstants.cs

```diff
@@ -18,6 +19,15 @@ public static class NodeConstants
         public const string YarnStartCommand = "yarn run start";
         public const string YarnCommand = "yarn";
         public const string HugoCommand = "hugo";
+        public const string LernaCommand = "lerna";
+        public const string LernaVersionCommand = "echo Using Npm version: && lerna --version";
+        public const string LernaInitCommand = "lerna init";
+        public const string LernaBootstrapCommand = "lerna bootstrap";
+        public const string InstallLernaCommandNpm = "npm install --global lerna";
```

Comment:
Does this have to be `global`?

Reply:
In its documentation, this is the command to install lerna tool. I didn't try installing it without `global`, I'll try it and see. Just wondering anything not good with `global`?

Reply:
@qianz2 we may not have permission to install it globally in appservice build image .. you can try this locally with oryxtests/build image and see if you are having any permission issue or anything as such while building a lerna app